index.html

```diff
@@ -1,8 +1,100 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang = "en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset = "UTF-8">
+    <meta name = "viewport" content = "width = device-width, initial-scale = 1">
     <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel = "stylesheet" href = "css/style.css">
   </head>
+
+  <body>
+    <h1 id = 'title'>freeCodeCamp Survey Form</h1>
+    <p>Thank you for taking the time to help us improve the platform</p>
+
+    <div>
+      <form id = 'survey-form'>
+        <div>
+          <label for = "name" style = "color: red">Name</label><br>
```

Comment:
- [] If you are trying to show this piece of info is required by styling the color of the text to the color red, you can easily add 'required property in your input tags. You have more than one red text in your code. Please consider applying this for all of them. Look this example: `<input class = "large" type = "text" id = "name" placeholder = "enter your name here please" required> ` .